src/components/app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import BrowserRouter from 'react-router-dom/BrowserRouter';
+import { renderRoutes } from 'react-router-config';
+
+import routes from './routes';
+
+const App = () => {
+    return (
+        <BrowserRouter>
+            {renderRoutes(routes)}
```

Comment:
Interesting. I don't think this is the intention with react router 4 but don't worry about that for now